setup.py

```diff
@@ -47,6 +47,8 @@
         "rtree>=1.2.0",
         "graphviz>=0.20",
         "matplotlib>=3.0.0",
+        "lxml>=5.3.0",
+        "pillow~=10.2.0",
```

Comment:
Yeah ideally this isn't even here at all -- but it's due to some really annoying isaac behavior where somehow the isaac shipped pip version of pillow (which is exactly 10.2) aliases your already-imported PIL version, resulting in "Can't find variable X" at runtime because somehow a function from the non isaac version of PIL (e.g.: 11.0) calls a source file from the pip-prebundle version from isaac.

I've run into this issue multiple times before and couldn't figure out a way to get this fixed, so this seemed like the easiest solution. Definitely open to a proper fix if you know of one (that ideally doesn't require deleting the isaac PIL version)